### DISCUSSION TITLE
Retrieve column values based on a criteria

### DISCUSSION THREAD

**Discussion by BismaIftekhar** [Budibase/budibase#11516] [Help]
Hello everyone,
Can anyone help with this please?

So, there is one public form and the source of it is a Budibase internal table. This table is called "Appointments-Form"  . If a row is created in this table(when someone fills the form) then based on the value of the column "Appointments" (based on the city name), it should return the corresponding email in another column called Site_Email 

Please note that the cities(London, Berlin, Paris,Amsterdam, Agra) are fixed and have their corresponding emails.
For reference,

![appointments_column_budibase](https://github.com/Budibase/budibase/assets/96243721/f8792740-cc9d-4ffc-9d73-16ee5117ac95)

Appreciate any help
Thank you!

**Comment by melohagan** [comment]
Hey @BismaIftekhar 

I'm not fully understanding the use-case. What have you tried so far?

**Reply by BismaIftekhar** [reply]
Thank you for your prompt response. 

So the purpose is to be able to send the form to the relevant site for which the appointment was created.  I thought of achieving this by creating a "site_email" column that returns the email in this column based on if the "Appointments" contains Paris then return "Paris@example.com" so that I can then use the trigger row automation to send the email to the site of Paris only lets say that a new appoitnment has been created with xyz details. 

I hope I was able to explain it the use case. Or is ther any other way to achieve this?

**Reply by melohagan** [reply]
So something like this?

1. User creates a new appointment, which includes a location city name
2. Based on the chosen city, lookup the email address table, and send an email with a summary of the new appointment

**Reply by BismaIftekhar** [reply]
The first point is correct. There is no seperate email address table though. So,
One main table with column "appointments" that includes the cities(hundreds of appoinments with cities name so 12.02.2023-23-02.2024 in Paris). So maybe a code that says if "appointmnets" includes Paris then return paris@example.com in the other column"site_email". This was my idea but cannot achieve it. Maybe there is another way to achieve it?
I can send you the app if its still not clear.
